src/pages/patientView/mutationalSignatures/MutationalSignaturesContainer.tsx

```diff
@@ -51,50 +126,247 @@ export default class MutationalSignaturesContainer extends React.Component<
         );
     }
 
+    public toggleButton(scale: AxisScale, onClick: () => void) {
+        return (
+            <button
+                className={classNames(
+                    {
+                        'btn-secondary': this.selectedScale === scale,
+                        'btn-outline-secondary': this.selectedScale !== scale,
+                    },
+                    'btn',
+                    'btn-sm',
+                    'btn-axis-switch'
+                )}
+                data-test={`AxisScaleSwitch${scale}`}
+                style={{
+                    lineHeight: 1,
+                    cursor:
+                        this.selectedScale === scale ? 'default' : 'pointer',
+                    fontWeight:
+                        this.selectedScale === scale ? 'bolder' : 'normal',
+                    color: this.selectedScale === scale ? '#fff' : '#6c757d',
+                    backgroundColor:
+                        this.selectedScale === scale ? '#6c757d' : '#fff',
+                    borderColor: '#6c757d',
+                }}
+                onClick={onClick}
+            >
+                {scale}
+            </button>
+        );
+    }
+    @observable yAxisLabel: string = 'Mutational count (value)';
+
+    @action.bound
+    private handlePercentClick() {
+        this.selectedScale = AxisScale.PERCENT;
+        this.yAxisLabel = 'Mutational count (%)';
+    }
+
+    @action.bound
+    private handleCountClick() {
+        this.selectedScale = AxisScale.COUNT;
+        this.yAxisLabel = 'Mutational count (value)';
+    }
+
+    @computed get getDataForGraph(): IMutationalCounts[] {
+        if (this.selectedScale == AxisScale.PERCENT) {
+            return getPercentageOfMutationalCount(
+                this.mutationalSignatureCountDataGroupedByVersionForSample
+            );
+        } else {
+            return this.mutationalSignatureCountDataGroupedByVersionForSample;
+        }
+    }
+
+    @autobind
+    private getSvg() {
+        if (document.getElementById('mutationalBarChart')) {
```

Comment:
look up React reference https://legacy.reactjs.org/docs/refs-and-the-dom.html to handle dom reference